BREAKING_CHANGES.md

```diff
@@ -4,6 +4,14 @@ NOTE: many style-related COMPONENT_IDs were renamed for consistency based
 on the CSS-in-JS restructure. All component-level theming customizations
 should be re-checked for ID naming accuracy.
 
+## @zendeskgarden/react-notifications
+
+- Boolean props for this package is updated to align with `isBoolean` naming. The following are
```

Comment:
Nit: "are updated..."

Reply:
The markdown linter asked to drop this line at 100 characters. The sentence continues below this line, but let me know if the sentence needs to be changed for different reason.

Reply:
Reviewing this line, I think you might have meant adding the word "updated" 👀.

Reply:
@rossmoody, did you think adding the word provides clarity (or maybe some other benefit)? I went ahead and added the word. 

I updated

> - Boolean props for this package is updated to align with `isBoolean` naming. The following are prop name changes for the `<Well>` component:
>
>   - `floating` -> `isFloating`
>   - `recessed` -> `isRecessed`

to

> - Boolean props for this package is updated to align with `isBoolean` naming. The following are updated prop name changes for the `<Well>` component:
>
>   - `floating` -> `isFloating`
>   - `recessed` -> `isRecessed`

Reply:
Third times a charm. Updated to:

> - Boolean props for this package are updated to align with `isBoolean` naming. The following are  prop name changes for the `<Well>` component: